test/request/component-models/LoggingComponentModel.spec.ts

```diff
@@ -10,6 +10,307 @@ import * as chai from 'chai';
 
 const should = chai.should();
 
+describe('LogMessageModel', () => {
```

Comment:
I added tests for back-to-back spans as well as when the message represents one entirely-replaced span.  I already have tests for spans at the very beginning and end of the message, so that should be covered.